packages/webapp/src/utils/constants.js

```diff
@@ -3,6 +3,7 @@ import { NODE_ENV, AZURE_FUNCTION_APP_URL } from './config.js'
 import lojConstants from './loj-constants.js'
 import creditsEstimationConstants from './credits-estimation-constants.js'
 import disabledRoutesContants from './disabled-routes-constants.js'
+import creditsPurchaseConstants from './credits-purchase-constants.js'
```

Comment:
Don't do this, for now import credits purchase constants (routes, views, redis keys) directly from `utils/credits-purchase-constants.js`